spec/models/claim_spec.rb

```diff
@@ -905,14 +905,15 @@
   end
 
   describe "Early Career Payments claim" do
```

Comment:
These specs made it in because the previous PR was rubber stamped but actually they don't look right and should be somewhere else.

I'm all for leveraging debt effectively but that's not what we're doing here.

Let's pair for a few minutes and get this sorted now rather than let it get out of hand and slow us down even more later on.

Reply:
Sure thing

Reply:
This is what we have just spent time on fixing so hopefully this is all good now from our joint pairing session